spring-cloud-dataflow-docs/src/main/asciidoc/spring-boot-3x.adoc

```diff
@@ -20,8 +20,6 @@ The naming of the metrics registry-specific properties differ as follows:
 * `2.x`: `management.metrics.export.prometheus.enabled=true`
```

Comment:
Do we want to keep the spring-boot-3x.adoc?  It contains both boot 3 and  boot 2 instructions on prometheus?
We really don't need the boot2 docs anymore do we?